nymea-app/ui/mainviews/WallboxView.qml

```diff
@@ -0,0 +1,170 @@
+                    width: swipeView.width
+                    height: swipeView.height
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    color: "green"
+
+                    gradient: Gradient {
+                        GradientStop { position: 0; color: "#07203F" }
+                        GradientStop { position: 1; color: "#01092A" }
+                    }
+
+                    GridLayout {
+                        anchors.fill: parent
+                        anchors.margins: app.margins
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        columns: 1
+                        rowSpacing: app.margins
+
+                        Text {
+                            anchors.horizontalCenter: parent.horizontalCenter
+                            text: thing.name
+                            color: "white"
+                        }
+
+                        Switch {
+                            anchors.horizontalCenter: parent.horizontalCenter
+                            id: powerSwitch
+                            checked: powerState.value
+                            onClicked: {
+                                root.setPower(powerState, checked)
+                            }
+                        }
+
+                        Text {
+                            Layout.alignment: parent.horizontalCenter
+                            anchors.horizontalCenter: parent.horizontalCenter
+                            text: maxChargingSlider.value + " " + qsTr("Ampere")
+                            color: "white"
+                        }
+
+                        ThrottledSlider {
+                            id: maxChargingSlider
+                            Layout.fillWidth: true
+                            Layout.leftMargin: parent.width * .05
+                            Layout.rightMargin: parent.width * .05
+                            from: 6
+                            to: 80
+                            stepSize: 1
+                            value: maxChargingCurrentState.value / 1000
+                            onMoved: {
+                                root.setMaxChargingCurrent(maxChargingCurrentState, value)
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    PageIndicator {
+        id: pageIndicator
+        count: swipeView.count
+        currentIndex: swipeView.currentIndex
+        interactive: true
+        anchors.bottom: parent.bottom
+        anchors.horizontalCenter: parent.horizontalCenter
+    }
+
+    EmptyViewPlaceholder {
+        anchors.centerIn: parent
+        width: parent.width - app.margins * 2
+        visible: !engine.thingManager.fetchingData && wallboxDevices.count == 0
+        title: qsTr("There are no wallbox devices set up.")
+        text: qsTr("Connect your wallbox devices in order to control them from here.")
```

Comment:
hmm, I suppose "wallboxes" would be better than "wallbox devices", wdyt?

Reply:
Adjusted.